core/esmf-aspect-model-resolver/src/main/java/org/eclipse/esmf/aspectmodel/resolver/AspectModelResolver.java

```diff
@@ -123,7 +127,7 @@ public Try<VersionedModel> resolveAspectModel( final ResolutionStrategy resoluti
     * @return the resolved model on success
     */
    public Try<VersionedModel> resolveAspectModel( final ResolutionStrategy resolutionStrategy, final InputStream inputStream ) {
-      return TurtleLoader.loadTurtle( inputStream ).flatMap( model -> resolveAspectModel( resolutionStrategy, model ) );
+      return TurtleLoader.loadTurtle( inputStream ).flatMap( model -> resolveAspectModel( FileSystemStrategy.DefaultNamespace.withDefaultNamespace(resolutionStrategy, model), model ) );
```

Comment:
Adjust code style: There should be spaces inside the brackets. Please also check other locations in this PR.